spacemesh/v2alpha1/layer.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+import "spacemesh/v2alpha1/block.proto";
+
+package spacemesh.v2alpha1;
+
+message Layer {
+  oneof versioned {
+    BlockV1 v1 = 1;
+  }
+}
+
+message LayerV1 {
+  uint32 number = 1; // layer number - not hash - layer content may change
+  enum LayerStatus {
+    LAYER_STATUS_APPLIED = 0; // applied by hare
```

Comment:
Every protobuf enum should have an `_UNSPECIFIED` element. There needs to be a way to distinguish whether or not this has been explicitly set, and it's the only way to do that. I'm fine with the names `APPLIED` and `VERIFIED` for the other statuses.